vms/shift/templates/shift/edit_hours.html

```diff
@@ -1,6 +1,22 @@
 {% extends "vms/base.html" %}
 
 {% block content %}
+<script type="text/javascript">
```

Comment:
@vubo I also checked we do not have any such file.

You need to add your JS file in the static files folder, which is [here](https://github.com/systers/vms/tree/develop/vms/vms/static). Create a folder there for JS files. Then edit your settings.py. [This](https://docs.djangoproject.com/en/1.8/howto/static-files/) will help you to move ahead.


Let me know if you face any issues.